src/payout-picker/StakeCard.js

```diff
@@ -56,7 +56,6 @@ export default class StakeCard extends Component {
                     min={0.35}
                     max={100000}
                     step="0.01"
-                    list="amounts"
```

Comment:
Oh didn't know you fix that . Its showing dropdown in a supposed textbox control which are not necessary. Nonetheless, I think the list params shouldn't be provided there or should it?